### PR TITLE
translating the validators for european portuguese language

### PR DESCRIPTION
**Pull request by brunosinister**
| Q             | A
| ------------- | ---
| Branch?       | 5.x for features / 4.4 or 5.1 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| Deprecations? | yes/no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 1]
```diff
@@ -8,12 +8,134 @@
...
+                <target>O campo oculto é inválido.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Por favor, informe um inteiro.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Por favor seleccione um idioma válido.</target>
+            </trans-unit><trans-unit id="113">
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
            </trans-unit>
            <trans-unit id="113">
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 2]
```diff
@@ -8,12 +8,134 @@
...
+                <source>Please select a valid locale.</source>
+                <target>Por favor, seleccione um locale válido.</target>
+            </trans-unit><trans-unit id="114">
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
            </trans-unit>
            <trans-unit id="114">
```

**Review by Nyholm — COMMENTED** [review]
Great! Thank you. I suggested some minor formatting fixes. 

I've got two questions. 
1. Is this Portuguese or European Portuguese? There is another file for European Portuguese specifically. 
2. Is it "selecione" or "seleccione"?

**Comment by Nyholm** [comment]
Could @hugovms and @adielcristo have a look at this too?

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 3]
```diff
@@ -8,12 +8,134 @@
...
+                <source>The checkbox has an invalid value.</source>
+                <target>O checkbox possui um valor inválido.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Por favor, informe um endereço de email válido.</target>
+            </trans-unit>
+            <trans-unit id="126">
+                <source>Please select a valid option.</source>
+                <target>Por favor seleccione uma opção válida.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf:129]: Minor fix:
Por favor**,** seleccione uma opção válida.

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 4]
```diff
@@ -8,12 +8,134 @@
...
+                <source>Please enter a valid URL.</source>
+                <target>Por favor, informe uma URL válida.</target>
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Por favor, informe um termo de busca válido.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>Por favor, provide a valid phone number.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf:113]: Should be translated:
Por favor, informe um número de telefone válido.

Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>Por favor, informe um número de telefone válido.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf (hunk 3 above)
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>Por favor, seleccione uma opção válida.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 5]
```diff
@@ -8,12 +8,134 @@
...
+                <source>Please enter a percentage value.</source>
+                <target>Por favor, informe um  valor percentual.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Os valores não correspondem.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Por favor, informe uma hora válido.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>Por favor, informe uma hora válida.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 6]
```diff
@@ -8,12 +8,134 @@
...
+                <source>Please enter a number.</source>
+                <target>Por favor, informe um número.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>A palavra-passe é inválida.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Por favor, informe um  valor percentual.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>Por favor, informe um valor percentual.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 7]
```diff
@@ -8,12 +8,134 @@
             </trans-unit>
             <trans-unit id="29">
                 <source>The uploaded file was too large. Please try to upload a smaller file.</source>
-                <target>O arquivo enviado é muito grande. Por favor, tente enviar um ficheiro mais pequeno.</target>
+                <target>O ficheiro enviado é muito grande. Por favor, tente enviar um ficheiro menor.</target>
             </trans-unit>
             <trans-unit id="30">
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
                 <target>O token CSRF é inválido. Por favor submeta o formulário novamente.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>O token CSRF é inválido. Por favor, submeta o formulário novamente.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 8]
```diff
@@ -8,12 +8,134 @@
...
+                <source>The hidden field is invalid.</source>
+                <target>O campo oculto é inválido.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Por favor, informe um inteiro.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Por favor seleccione um idioma válido.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>Por favor, seleccione um idioma válido.</target>
```

**Comment by nicolas-grekas** [comment]
Thank you @brunosinister.

**Pull request merged**